examples/seq2seq/run_seq2seq.py

```diff
@@ -351,8 +374,15 @@ def main():
     )
 
     # Set decoder_start_token_id
-    if model.config.decoder_start_token_id is None and isinstance(tokenizer, MBartTokenizer):
-        model.config.decoder_start_token_id = tokenizer.lang_code_to_id[data_args.target_lang]
+    if model.config.decoder_start_token_id is None and isinstance(tokenizer, (MBartTokenizer, MBartTokenizerFast)):
+        assert (
+            data_args.target_lang is not None and data_args.source_lang is not None
+        ), "mBart requires --target_lang and --source_lang"
+        if isinstance(tokenizer, MBartTokenizer):
+            model.config.decoder_start_token_id = tokenizer.lang_code_to_id[data_args.target_lang]
+        else:
+            model.config.decoder_start_token_id = tokenizer.convert_tokens_to_ids(data_args.target_lang)
```

Comment:
Thanks for putting that logic back. @patil-suraj @patrickvonplaten since you better than me, shouldn't this be done by `set_tgt_lang_special_tokens` inside mBART?

Reply:
Can we make a priority on merging this, so that I could finish porting the other tests? And this can be dealt with separately? 

I can open an issue so that it doesn't fall between the cracks.

Thank you!

Reply:
`set_tgt_lang_special_tokens` is a method on `MBartTokenizer` so this needs to be done outside of the model.

Also, IMO `MBartTokenizerFast` should also have the `lang_code_to_id` attribute, not sure why it was treated differently.

Reply:
OK, I decided to go ahead and port the other scripts instead of waiting for merging of the first set. 

Had to make some more fixes in the script while at it. 

So no rush.

Reply:
Agree with @patil-suraj here -> IMO `MBartTokenizerFast` should have the `lang_code_to_id` so that we don't need an if-else here...but this is not necessarily the responsibility of this PR, so it's fine for me as it is

Reply:
Maybe for models such as MBart that don't have an unique `decoder_start_token_id`, we should maybe in the future just leave `config.decoder_start_token_id=None` and then throw an error / warning when generating

Reply:
Setting it manually here `model.config.decoder_start_token_id` is the correct thing to do here, but not super pretty IMO -> it would be better to incentivise the user to either insert at init or when calling `generate` But because of backward comp we can't really change it now for mBART anyways...